### PR TITLE
O2105 wave 1

### PR DESCRIPTION
**Pull request by PeterAlabaster**
Adds statement of account report

### CODE REVIEW

File: UK_Reports/models/res_company.py [hunk 1]
```diff
@@ -58,6 +58,11 @@ class ResCompany(models.Model):
 	delivery_tandc = fields.Text("Delivery terms & Conditions")
 	purchase_tandc = fields.Text("Purchase Order terms & Conditions")
 	rfq_tandc = fields.Text("Request for Quotation terms & Conditions")
+	statement_of_accounts_tandc = fields.Text("Statement of Account Message")
+	statement_of_accounts_account_filter = fields.Many2many(
+		'account.account',
+		string="Statement of Account from",
+		help="The account which will be used on the Statement of Account report")
```
Comment (james-opusvl) [inline UK_Reports/models/res_company.py]:
```suggestion
		help="The accounts which will be used on the Statement of Account report")
```

File: UK_Reports/reports/statement_of_account_report.xml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+					<span t-esc="context_timestamp(datetime.datetime.now()).strftime('%d-%m-%Y')"/>   
+					<br/>
+					<t t-if="o.ref">
+						<strong>Customer Ref</strong>
+						<span t-field="o.ref"/>
+						<br/>
+					</t>
+					<t t-if="o.user_id">
+						<strong>Account Manager</strong>
+						<span t-field="o.user_id.name"/>
```
Comment (james-opusvl) [inline UK_Reports/reports/statement_of_account_report.xml:24]: Is this always going to be right? ie from the account manager group

**Review by james-opusvl — COMMENTED** [review]
See inline comments.

I only commented on 1 plural, but if considered it may occur in other places.

File: UK_Reports/reports/statement_of_account_report.xml (hunk 2 above)
Comment (PeterAlabaster) [reply UK_Reports/reports/statement_of_account_report.xml:24]: Narender said yes

**Pull request merged**